app/models/grda_warehouse/synthetic/assessment.rb

```diff
@@ -34,9 +34,10 @@ def self.hud_sync
       create_hud_assessments
 
       # Clean up orphans in HUD table
-      GrdaWarehouse::Hud::Assessment.
-        where(synthetic: true).
-        where.not(AssessmentID: select(:hud_assessment_assessment_id)).
+      assessment_source.
+        synthetic.
+        # ActiveRecord doesn't work with a simple select since it returns nil in the ID field, must use Arel.sql
```

Comment:
You moved to `project` after you wrote this...
```suggestion
        # ActiveRecord doesn't work with a simple select since it returns nil in the ID field
```